reliability-engineering/pipelines/build-govsvc-docker.yml

```diff
@@ -86,3 +107,17 @@ jobs:
     params:
       build: govsvc-octodns-git/reliability-engineering/dockerfiles/govsvc/octodns
     get_params: {skip_download: true}
+- name: build-govsvc-awsc
+  serial: true
+  plan:
+  - in_parallel:
+    - get: govsvc-awsc-git
+      trigger: true
+    - get: golang-1.12-image
+      trigger: true
+      params: {save: true}
+  - put: govsvc-awsc-image
+    params:
+      build: govsvc-awsc-git/reliability-engineering/dockerfiles/govsvc/awsc
+      load_bases: [ golang-1.12-image ]
```

Comment:
@Krenair Is this the `load_bases` change you wanted?

Reply:
yeah